packages/@aws-cdk-containers/ecs-service-extensions/lib/extensions/container.ts

```diff
@@ -93,6 +108,30 @@ export class Container extends ServiceExtension {
       containerProps = hookProvider.mutateContainerDefinition(containerProps);
     });
 
+    // If no observability extensions have been added to the service description, we can configure
+    // default awslogs log driver
+    if (!containerProps.logging) {
+      // Create a log group for the service, if one is not provided by the user
+      if (!this.logGroup) {
+        this.logGroup = new awslogs.LogGroup(this.scope, `${this.parentService.id}-logs`, {
+          logGroupName: `${this.parentService.id}-logs`,
+          removalPolicy: cdk.RemovalPolicy.DESTROY,
+          retention: awslogs.RetentionDays.ONE_MONTH,
+        });
+      }
+
+      containerProps = {
```

Comment:
Is this equal to `containerProps.logging = ...`?

Reply:
Yes it is! But as `logging` is a `read-only` property, we cannot update it’s value this way. So we need to create a new object with the updated values.